platform/windows/detect.py

```diff
@@ -441,6 +460,16 @@ def configure_msvc(env, vcvars_msvc_config):
             LIBS += ["vulkan"]
 
     if env["d3d12"]:
+        # Check whether we have d3d12 dependencies installed.
+        if not os.path.exists(env["mesa_libs"]) or not os.path.exists(env["dxc_path"]):
```

Comment:
A bit too late but DXC is not required for building. Mesa libs is the only requirement.